packages/procaptcha/src/modules/Manager.ts

```diff
@@ -122,11 +118,9 @@ export const Manager = (
     const start = async () => {
         try {
             if (state.loading) {
-                console.log('Procaptcha already loading')
```

Comment:
would it be better to move these to console.debug()s?

also, I believe webpack can strip out the console logs so we don't have to ourselves

Reply:
Totally. Yeah, we can strip console logs using webpack and we should do that.

Reply:
discussed on slack